### PR TITLE
WiP: PoC codeceptJS & WebDriverIO

### PR DESCRIPTION
**Pull request by evkotdrsy**
Skeleton of the possible framework

### CODE REVIEW

File: config/staging.setup.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+    "helpers": {
+      "WebDriver": {
+        "url": "https://www.staging.outdoorsy.com/"
+      },
+      "REST": {
+        "endpoint": "https://exp.outdoorsy.com/api/v1/"
```
Comment (ryanpatrickcook) [inline config/staging.setup.json:7]: https://exp.staging.outdoorsy.com/

File: codecept.conf.js [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+                '--start-fullscreen'
+              ]
+          }
+        }
+    },
+    ChaiWrapper: {
+      require: 'codeceptjs-chai'
+    },
+    REST: {
+        endpoint: 'https://exp.outdoorsy.com/api/v1/',
```
Comment (ryanpatrickcook) [inline codecept.conf.js:49]: exp.staging

**Review by jayoutdoorsy — COMMENTED** [review]
Should also set up eslint/prettier that follows similar standards as `outdoorsy-renter` and/or next/mono

File: config/staging.setup.json (hunk 1 above)
Comment (evkotdrsy) [reply config/staging.setup.json:7]: fixed

File: codecept.conf.js (hunk 2 above)
Comment (evkotdrsy) [reply codecept.conf.js:49]: fixed

**Comment by evkotdrsy** [comment]
@jayoutdoorsy fully agree, just we plan to use `tslint` on CI level, any configs we could borrow?

**Review by gholmesoutdoorsy — APPROVED** [review]

**Pull request merged**